docs/source/api/user-guide/plugins/measurement.rst

```diff
@@ -24,42 +24,47 @@ This example mocks the typical API of the OpenDP library to make the *most priva
 
   .. tab-item:: Python
 
-    .. code:: python
+    .. code:: pycon
 
         >>> def make_base_constant(constant):
         ...     """Constructs a Measurement that only returns a constant value."""
         ...     def function(_arg: int):
         ...         return constant
-        ... 
         ...     def privacy_map(d_in: int) -> float:
         ...         return 0.0
-        ...
         ...     return dp.m.make_user_measurement(
         ...         input_domain=dp.atom_domain(T=int),
         ...         input_metric=dp.absolute_distance(T=int),
         ...         output_measure=dp.max_divergence(),
         ...         function=function,
         ...         privacy_map=privacy_map,
-        ...         TO=type(constant),  # the expected type of the output
+        ...         TO=type(
+        ...             constant
+        ...         ),  # the expected type of the output
         ...     )
+        ...
     
 The resulting Measurement may be used interchangeably with those constructed via the library:
 
 .. tab-set::
 
   .. tab-item:: Python
 
-    .. code:: python
+    .. code:: pycon
 
         >>> meas = (
-        ...     (dp.vector_domain(dp.atom_domain((0, 10))), dp.symmetric_distance())
+        ...     (
+        ...         dp.vector_domain(dp.atom_domain((0, 10))),
+        ...         dp.symmetric_distance(),
+        ...     )
         ...     >> dp.t.then_sum()
         ...     >> make_base_constant("denied")
         ... )
-        ...
         >>> meas([2, 3, 4])
         'denied'
-        >>> meas.map(1) # computes epsilon, because the output measure is max divergence
+        >>> meas.map(
+        ...     1
+        ... )  # computes epsilon, because the output measure is max divergence
```

Comment:
Stuff like this is unfortunate, and could be cleaned up by some manual work splitting lines.